packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -8,19 +8,15 @@ import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
 
 import DecentralandUtils from "./DecentralandUtils";
 
-import {
-  mineBlocks,
-  createSnapshot,
-  revertFromSnapshot,
-  confirmBalances,
-  etherFaucet,
-  erc20Faucet,
-  addressesAreEqual,
-  constants,
-  bigNumberify,
-  gasParams,
-  ProviderFactory,
-} from "tasit-action/dist/testHelpers/helpers";
+// Helpers
+import actionHelpers from "tasit-action/dist/testHelpers/helpers";
+global = Object.assign(global, actionHelpers);
```

Comment:
Is this a pattern that people are generally okay with? I think `global`s should be used pretty sparingly

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/237

Reply:
Thanks. Resolving